test/java/org/opendatakit/briefcase/reused/UncheckedFilesTest.java

```diff
@@ -0,0 +1,65 @@
+package org.opendatakit.briefcase.reused;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createFile;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createTempDirectory;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.deleteRecursive;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class UncheckedFilesTest {
+  private static Path tempDir;
+  private Path temp;
+
+
+  @Before
+  public void setUp() {
+    tempDir = createTempDirectory("briefcase");
+    temp = Paths.get(tempDir.toString() + "/test.txt");
+    createFile(temp);
+  }
+
+  @After
+  public void tearDown() {
+    deleteRecursive(tempDir);
+  }
+
+  @Test(expected = UncheckedIOException.class)
+  public void newInputStream_should_throw_exception() {
+    UncheckedFiles.delete(temp);
+    UncheckedFiles.newInputStream(Paths.get(tempDir.toString() + "/test.txt"));
+  }
+
+  @Test
+  public void closeInputStream_should_handle_null() {
+    UncheckedFiles.closeInputStream(null);
+  }
+
+  @Test(expected = IOException.class)
+  public void closeInputStream_should_close() throws IOException {
+    InputStream inputStream = UncheckedFiles.newInputStream(Paths.get(tempDir.toString() + "/test.txt"));
```

Comment:
```suggestion
    InputStream inputStream = UncheckedFiles.newInputStream(temp);
```
You can use `temp` here instead of building a new `Path` (also, remember to use `Path.resolve()` ;) )